src/components/layout/MainNavigation.tsx

```diff
@@ -0,0 +1,24 @@
+import Logo from './Logo.jsx'
+import { NavLink, Link } from 'react-router-dom'
+
+const MainNavigation = () => {
+    const activeClass = 'bg-primary text-green font-nav px-3 py-1 text-[30px] '
```

Comment:
static things go outside components

Reply:
when a component rerenders we do not want to let it work all these again, they can be cached by keeping outside the component

Reply:
Good idea. Also I think these can be reusable in some app. So maybe it's even worth moving them to a util file or inside a ts config for some projects. Anyway done.